src/components/dropdown-menu/DropdownMenu.scss

```diff
@@ -17,6 +17,50 @@
 
     z-index: $menu-z-index;
     margin-top: 5px;
-    perspective: 500px;
-    perspective-origin: center top;
+}
+
+@include breakpoint($medium-screen) {
+    .bdl-DropdownMenu--responsive {
+        &.dropdown-menu-element {
+            right: 0;
+            bottom: 0;
+            left: 0;
+            margin-top: 0;
+        }
+
+        &.dropdown-menu-enabled {
+            // cancels out the tether inline styling without having to set enabled=false
+            transform: none !important;
```

Comment:
Can you elaborate on the alternative here? ie setting `enabled=false`?

Reply:
This is a workaround to cancel out the styling in CSS. The tether library applies a transform override with `!important `which makes it impossible to override without specifying another `!important` override. This limited the JS changes I needed to make which I felt was cleaner than to have to use the media query library for that one change